source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
+     * of the driver collection object is used, if one is defined. Otherwise, the default collation of
+     * the collection server-side is used.
```

Comment:
Also mention the server version this can be used with. See, for example, storageEngine.